src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -4,17 +4,18 @@
     [react-native.core :as rn]
     [status-im2.common.scroll-page.view :as scroll-page]
     [status-im2.contexts.wallet.collectible.style :as style]
-    [status-im2.contexts.wallet.common.temp :as temp]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
 
 (defn header
-  [{:keys [name description collection-image]}]
+  [{:keys [name description collection-image-url]}]
   [rn/view {:style style/header}
    [quo/text
     {:weight :semi-bold
      :size   :heading-1} name]
-   [rn/view style/collection-container
-    [quo/collection-avatar {:image collection-image}]
+   [rn/view {:style style/collection-container}
+    [rn/view {:style style/collection-avatar-container}
+     [quo/collection-avatar {:image collection-image-url}]]
```

Comment:
should we add a `container-style` prop to this component? e.g so it can be 
```
   [rn/view {:style style/collection-container}
     [quo/collection-avatar {
     :container-style style/collection-avatar-container
     :image collection-image-url}]]
 ```

Reply:
`collection-avatar` inside doesn't have additional containers, only fast-image. Also so far it is used only here. So I'd postpone this change until we see that pattern of having external containers emerge.